new_course/1_programming_fundementals/debugging.md

```diff
@@ -122,11 +122,13 @@ def double_words_in_phrase(string)
   string_array.map { |word| word * 2}
   string_array.join(' ')
 end
+
+double_words_in_phrase("This is a test")
 ~~~
 
 When your code executes and gets to `binding.pry`, it will open an IRB-like session in your terminal. You can then use that session to check the values of anything within the scope of where you included `binding.pry`.
 
-Using Pry for debugging achieves the same outcome as `puts` debugging: it allows you to confirm the assumptions you have about particular parts of your code. If your code is complex, Pry will probably allow you to debug quicker thanks to its interactive runtime environment. In such scenarios, Pry will be easier to interact with than having to add `puts` statements everywhere and re-running your code each time.
+Using Pry for debugging achieves the same outcome as `puts` debugging: it allows you to confirm the assumptions you have about particular parts of your code. If your code is complex, Pry will probably allow you to debug quicker thanks to its interactive runtime environment. In such scenarios, Pry will be easier to interact with than having to add `puts` statements everywhere and re-running your code each time. Plus, you won't have to remember to go back and remove all those `puts` statements from your code when you are finished.
```

Comment:
I removed this addition since the implication is that you wouldn't have to go back and revise your code when using Pry, but you *will* need to go back and remove any `binding.pry` statements.

Reply:
That's a good point!